website/docs/reference/resource-configs/snowflake-configs.md

```diff
@@ -13,13 +13,15 @@ To-do:
 
 ## Iceberg table format <Lifecycle status="beta"/>
 
-The dbt-snowflake adapter supports the Iceberg table format. It is available for three of the Snowflake materializations: 
+The dbt-snowflake adapter supports the Iceberg table format. Prior to developing an Iceberg table with dbt, you need to [configure an external volume in Snowflake](https://docs.snowflake.com/en/user-guide/tables-iceberg-configure-external-volume).
+
+In dbt, Iceberg is available for three of the Snowflake materializations: 
 
 - [Table](/docs/build/materializations#table)
 - [Incremental](/docs/build/materializations#incremental)
-- [Dynamic](#dynamic-tables) 
+- [Dynamic](#dynamic-tables)
 
-For now, to create Iceberg tables, you must implement a [behavior flag](/reference/global-configs/behavior-changes) due to performance impact related to using Iceberg tables. Snowflake does not support `is_iceberg` on the `Show Objects` query, which dbt depends on for metadata.
```

Comment:
I removed the part about `SHOW OBJECTS` as it felt a little niche / tangental, but let me know if you disagree!

Reply:
Just saw this pop up - I disagree with this :) This is a dbt metadata performance impact rather than from using Iceberg tables. (also this helps snowflake customers advocate for us)